### ISSUE TITLE
ButtonView ignores colors

### ISSUE THREAD

**Issue by irskep** [irskep/clubsandwich#4]
It should use color_bg and color_fg as the colors for the label, and invert when selected.

**Comment by jainokaine** [comment]
I lifted this for easy color inversion if we exclusively use hexcode. Down side is some colors are downright invisible or difficult to see when contrasted. Perhaps the FG is when un-selected and when selected, it changes to a good Neutral color (like White, Grey, or Black depending on the color. No clue how to implement.)

```
def oppositeColor(color):
    if color[0] == '#':
        color = color[1:]
    rgb = (color[0:2], color[2:4], color[4:6])
    comp = ['%02X' % (255 - int(a, 16)) for a in rgb]
    bkcolor = "[bkcolor=#"
    bkcolor += ''.join(comp)
    bkcolor += ']'
    return bkcolor
```

**Comment by irskep** [comment]
Fixed by 8e74d51e79afbd639f5e3a8ac912cd731b66dc76
